google/cloud/spanner_v1/types/spanner.py

```diff
@@ -922,12 +930,36 @@ class CommitResponse(proto.Message):
         commit_timestamp (~.timestamp.Timestamp):
             The Cloud Spanner timestamp at which the
             transaction committed.
+        commit_stats (~.spanner.CommitResponse.CommitStats):
+            The statistics about this Commit. Not returned by default.
+            For more information, see
+            [CommitRequest.return_commit_stats][google.spanner.v1.CommitRequest.return_commit_stats].
     """
 
+    class CommitStats(proto.Message):
+        r"""Additional statistics about a commit.
+
+        Attributes:
+            mutation_count (int):
+                The total number of the mutations for the
+                transaction.
+            overload_delay (~.duration.Duration):
+                Length of time the commit was delayed due to
+                overloaded servers.
```

Comment:
I assume this is a generated file. If so it's fine, but if not, then remove the `overload_delay` references as we're not releasing that right now.